modules/wizard/js/languages-step.js

```diff
@@ -277,23 +277,23 @@ jQuery( document ).ready(
 					$( this ).find( '#dialog-language-flag' ).empty().prepend( $( '#lang_list' ).children( ':selected' ).data( 'flag-html' ) ); // phpcs:ignore WordPressVIPMinimum.JS.HTMLExecutingFunctions.prepend
 				},
 				buttons: [
-				{
-					text: pll_wizard_params.i18n_dialog_yes_button,
-					click: function( event ) {
-						confirmDialog( 'yes' );
-					}
+					{
+						text: pll_wizard_params.i18n_dialog_yes_button,
+						click: function( event ) {
+							confirmDialog( 'yes' );
+						}
 				},
-				{
-					text: pll_wizard_params.i18n_dialog_no_button,
-					click: function( event ) {
-						confirmDialog( 'no' );
-					}
+					{
+						text: pll_wizard_params.i18n_dialog_no_button,
+						click: function( event ) {
+							confirmDialog( 'no' );
+						}
 				},
-				{
-					text: pll_wizard_params.i18n_dialog_ignore_button,
-					click: function( event ) {
-						confirmDialog( 'ignore' );
-					}
+					{
+						text: pll_wizard_params.i18n_dialog_ignore_button,
+						click: function( event ) {
+							confirmDialog( 'ignore' );
+						}
 				}
 				]
```

Comment:
After correcting the indentation the closing curly brackets should be aligned to the opening ones